lightning/src/routing/router.rs

```diff
@@ -2122,6 +2122,20 @@ impl<'a> PaymentPath<'a> {
 		return result;
 	}
 
+	/// Gets the cost (fees plus scorer penalty in msats) of the path divided by the value we
+	/// can/will send over the path. This is also the heap score during our Dijkstra's walk.
+	fn get_cost_per_msat(&self) -> u128 {
+		let fee_cost = self.get_cost_msat();
+		let value_msat = self.get_value_msat();
+		debug_assert!(value_msat > 0, "Paths should always send more than 0 msat");
+		if fee_cost == u64::MAX || value_msat == 0 {
+			u64::MAX.into()
+		} else {
+			((fee_cost as u128) << 64) / value_msat as u128
```

Comment:
Can we extract the `64` const here and below and document why it was chosen? Otherwise it's not clear here unless you see the comment in the middle of the big macro below...